src/Provider/TranslationServiceProvider.php

```diff
@@ -137,7 +137,7 @@ public static function addResources(Application $app, $locale)
              * @var \SplFileInfo $fileInfo
              */
             foreach ($iterator as $fileInfo) {
-                $ext = Lib::getExtension((string) $fileInfo);
+                $ext = pathinfo($fileInfo, PATHINFO_EXTENSION);
```

Comment:
Wondered how many years until someone finally changed this line to the native function … mind youm the native one was only introduced in PHP 4.0.3 (yes, v4)